sdk/signalr/Microsoft.Azure.WebJobs.Extensions.SignalRService/azure-functions-signalrservice-extension.sln

```diff
@@ -0,0 +1,130 @@
+﻿
+Microsoft Visual Studio Solution File, Format Version 12.00
```

Comment:
I'd suggest renaming the solution to match the convention of other WebJobs solutions.